docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -289,6 +294,84 @@ spec:
+----
+<1> The `*packages*` list is the list of packages you want to install in your Container. Only include first-order dependencies here, i.e. the packages you want to install.
+<2> This should be a reference to a repo file, like those found in `/etc/yum.repos.d/`. This tells the Cachi2 where to find your rpm and its dependencies.
+
+. Copy any necessary yum/dnf repo files into your git repository. If you are using a fedora base image, that looks like:
+
++
+[source]
+----
+$ cp /etc/yum.repos.d/fedora.repo .
+----
+
+. Run the following command to resolve your `rpms.in.yaml` file and produce a `rpms.lock.yaml` file.
+
++
+[source]
+----
+$ rpm-lockfile-prototype --image YOUR_BASE_IMAGE rpms.in.yaml <1>
+----
+<1> The produced `rpms.lock.yaml` file will include only your requested dependency plus its transitive dependencies, minus any rpms that are already installed in the provided base image.
+
+. Go to the `.tekton` directory and find the `.yaml` files related to the `*pull request*` and `*push*` processes.
+. Configure the hermetic pipeline by adding the following parameters in both `.yaml` files:
+
++
+[source,yaml]
+----
+spec:
+    params:
+        -   ...
+        -   name: prefetch-input
+            value: '{"type": "rpm", "path": "."}' <1>
+----
+<1> The `*prefetch-input*` parameter specifies the path to the directory that has the lockfile. In this example, the `.` indicates that the rpm lockfile is in the repository root. Additionally, if you have multiple directories, you can provide the path to those directories in the JSON array format. For example, `[{"path": ".", "type": "rpm"}, {"path": "subpath/to/the/other/directory", "type": "rpm"}]`.
+
+. Additionally, pass an extra parameter to the `prefetch-dependencies` task to indicate that "dev package managers" should be enabled.
+
++
+[source,yaml]
+----
+spec:
+    tasks:
+        -   ...
+        -   name: prefetch-dependencies
+            params:
+                - ...
+                - name: dev-package-managers <1>
+                  value: "true"
+----
+<1> You won't find `dev-package-mangers` as a param on the `prefetch-dependencies` task. You have to add it, and set it to true. This is because Cachi2 hasn't declared stable support for rpm lockfile processing yet. It's new technology and the link:https://github.com/rpm-software-management/dnf5/issues/833[conversation] about which way forward in the dnf community is still ongoing.
+
+. Create a pull request by committing your changes to the repository of the component.
+. Review and merge the pull request.
+
+.Verification
+* From the {ProductName} *Applications* view, go to *Activity > Pipeline runs*.
+** Go to the pipeline run with *Build* in the *Type* column and confirm that the `prefetch-dependencies` stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.
+* From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
+
```

Comment:
Was something supposed to follow here?

Reply:
Ah, I was missing

```
//include::../partials/con_hermetic_verification.adoc[]
```

Adding that in now.